### ISSUE TITLE
Map screen

### ISSUE THREAD

**Issue by swiftcode** [devict/publicmeetings-ios#70]
Mapping a meeting location takes the following path:

Meetings Screen --> Meetings Detail Screen --> Apple Maps

In the current implementation, the Meetings Detail screen acts little more than a pass-through screen.  The segue back from Apple Maps hits the Meetings detail screen which doesn't contain any information (there were plans for it).  

I added code to dismiss the meeting detail screen when Apple Maps is loaded, but this is not a long-term solution.  The segue back works, but either Apple Maps should be called from the Meetings screen or the concept of the detail screen should be reexamined to see if an implementation for it is warranted. 

In summary:  Either come up with a plan for the meetings detail screen or launch maps from the main Meetings screen and remove the detail screen altogether.

**Comment by swiftcode** [comment]
Fixed by removing the defunct view controller and running the map code from MeetingsViewController.
